README.md

```diff
@@ -0,0 +1,25 @@
+# ayni #
+
+Ayni is a game about solidarity and cooperation, the game starts when a group of people start to form a cooperative to install pipes and Have water in a town.
+
+Ayuda mutua que se prestan las personas en el trabajo en Quechua (Mutual help provided by people at work in Quechua).
```

Comment:
@quozl Is this necessary?

Reply:
No, but I [asked for it not to be removed](https://github.com/sugarlabs/ayni-activity/pull/4#issuecomment-577467679), and so far the reason for removing it has been "i can't read that language".  We try to be inclusive.

Reply:
Ok